gems/aws-sdk-s3/spec/object/download_file_spec.rb

```diff
@@ -172,8 +172,10 @@ module S3
         end
 
         it 'raises an error when checksum validation fails on multipart' do
```

Comment:
I think it may be in the upload stream spec and not this one, at least based on this failure: https://github.com/aws/aws-sdk-ruby/actions/runs/5754207155/job/15599029463

We should take a separate task to update that.